docs/reviewer-guide.md

```diff
@@ -121,4 +121,5 @@ The lowest bar for acceptable golang **coding standards** (anti-patterns, coding
 
 The lowest **testing bar** to pass:
 * New code requires new unit tests.
-* New features and bug fixes require at least on e2e test (the core case must be tested).
+* New features at least one e2e test (only the core case must be tested)
```

Comment:
What do you mean by `(only the core case must be tested)`?

Reply:
I actually just copied it from the prev version.

I've reworded it.